vignettes/example1.Rmd

```diff
@@ -185,11 +188,11 @@ map_estimate(posteriors$Petal.Length)
 
 ```{r message=FALSE, warning=FALSE}
 ggplot(posteriors, aes(x = Petal.Length)) +
-  geom_density(fill = "orange") +
+  geom_density(fill = "wheat") +
   # The mean in blue
-  geom_vline(xintercept=mean(posteriors$Petal.Length), color="blue", size=1) +
```

Comment:
for these vline colors I think it's better to keep raw colours (blue and red) as it is how we refer to them after

Reply:
Ahhh I missed this, good spotting.